flows/epsfbuilder/epsfbuilder.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+Photutils hack for EPSF building
+
+.. codeauthor:: Simon Holmbo <simonholmbo@phys.au.dk>
+"""
+import time
+
+import numpy as np
+
+#from scipy.spatial import cKDTree
+from scipy.interpolate import griddata
+
+import photutils.psf
+
+
+class EPSFBuilder(photutils.psf.EPSFBuilder):
```

Comment:
Same thing as for the "WCS" object... Lets give it a new name so there is no confusion with the original `EPSFBuilder`... `FlowsEPSFBuilder` maybe?

Reply:
Since we're just copying and extending it, I think we should keep the same name. Then in the future if we move back to photutils EPSFBuilder, we don't have to refactor the code.